rs/sns/governance/canister/governance.did

```diff
@@ -248,6 +248,15 @@ type GetMetadataResponse = record {
   description : opt text;
 };
 
+type GetSnsStatusRequest = record {
+  time_window : opt nat64;
+}
+
+type GetSnsStatusResponse = record {
+  recent_proposals : opt nat64;
```

Comment:
Nit

```suggestion
  num_recent_proposals : opt nat64;
```

Reply:
My reasoning: Later on, we might add something like `recent_proposals : opt vec ProposalId;`